CK3toEU4/Data_Files/configurables/culture_map.txt

```diff
@@ -99,12 +99,12 @@ link = { eu4 = neapolitan ck3 = italian ck3 = cisalpine ck3 = sicilian tech = we
 link = { eu4 = piedmontese ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = piedmont_area region = savoy_dauphine_area region = provence_area region = romandie_area }
 link = { eu4 = romagnan ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = emilia_romagna_area }
 link = { eu4 = sardinian ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = corsica_sardinia_area }
-link = { eu4 = sicilian ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = sicily_area }
+link = { eu4 = sicilian ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = sicily_area region = western_sicily_area }
 link = { eu4 = tuscan ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = tuscany_area }
 link = { eu4 = venetian ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = venetia_area region = carinthia_area region = tirol_area }
 link = { eu4 = dalmatian ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx region = east_adriatic_coast_area region = croatia_area region = albania_area }
-link = { eu4 = sicilian ck3 = sicilian tech = western gfx = westerngfx } # Fallback
-link = { eu4 = italian ck3 = italian ck3 = cisalpine tech = western gfx = westerngfx } #Fallback
+link = { eu4 = sicilian ck3 = sicilian tech = western gfx = westerngfx } #Fallback
+link = { eu4 = italian ck3 = italian ck3 = cisalpine ck3 = sicilian tech = western gfx = westerngfx } #Fallback
```

Comment:
No need for Sicilian to have Italian as fallback because of the very line above this one

Reply:
Theoretically yes, but I like my super fallbacks.